back/boxtribute_server/graph_ql/definitions/protected/mutations.graphql

```diff
@@ -33,6 +33,9 @@ type Mutation {
   createCustomProduct(creationInput: CustomProductCreationInput): CreateCustomProductResult
   editCustomProduct(editInput: CustomProductEditInput): EditCustomProductResult
   deleteProduct(id: ID!): DeleteProductResult
+  enableStandardProduct(enableInput: StandardProductEnableInput): EnableStandardProductResult
+  editStandardProductInstantiation(editInput: StandardProductInstantiationEditInput): EditStandardProductInstantiationResult
+  disableStandardProduct(id: ID!): DisableStandardProductResult
```

Comment:
pls make clear that the id is the id in the products table and not the id  in standard_products